easybuild/easyconfigs/p/pftools/pftools-V3-foss-2016a.eb

```diff
@@ -0,0 +1,21 @@
+easyblock='ConfigureMake'
+
+name = 'pftools'
+version = 'V3'
```

Comment:
@RvDijk seems like `pftoolsV3` is the software name?

and since the tarball isn't properly versioned, using a datestamp as version and add a checksum for the source tarball?